Sources/StreamChat/Database/StreamChatModel.xcdatamodeld/StreamChatModel.xcdatamodel/contents

```diff
@@ -99,6 +117,13 @@
         <relationship name="channel" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="ChannelDTO" inverseName="reads" inverseEntity="ChannelDTO"/>
         <relationship name="user" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="UserDTO" inverseName="channelReads" inverseEntity="UserDTO"/>
     </entity>
+    <entity name="CommandDTO" representedClassName="CommandDTO" syncable="YES">
+        <attribute name="args" attributeType="String"/>
```

Comment:
Is there any reason why `args` is the only field that does not have default value?

Reply:
It is a mistake, thanks! 🙏